.appveyor.yml

```diff
@@ -54,10 +54,10 @@ build_script:
 test_script:
   # TODO: need an equivalent of TRAVIS_COMMIT_RANGE
   # - GIT_CHECK_EXCLUDE="./vendor" TRAVIS_COMMIT_RANGE="${TRAVIS_COMMIT_RANGE/.../..}" C:\MinGW\bin\mingw32-make.exe dco
-  - bash.exe -lc "export PATH=/c/tools/mingw64/bin:/c/gopath/src/github.com/containerd/containerd/bin:$PATH ; mingw32-make.exe coverage root-coverage"
-  # - bash.exe -elc "export PATH=/c/tools/mingw64/bin:/c/gopath/src/github.com/containerd/containerd/bin:$PATH ; mingw32-make.exe integration"
+  - bash.exe -lc "export CGO_ENABLED=1 PATH=/c/tools/mingw64/bin:/c/gopath/src/github.com/containerd/containerd/bin:$PATH ; mingw32-make.exe coverage root-coverage"
```

Comment:
I'm confused by this. Why do we still set CGO_ENABLED=1 here and include mingw64 in the path?

Reply:
The Race Detector relies on CGO, and we put `-race` into `TEST_FLAGS` on Windows/AMD64 via _Makefile.windows_. See f28c191570e1152580bb40e83b49f034d0159959.

I also have a version of this change that removes the `-race` flag, and hence removes MingW entirely from this build, in favour of GNU Make. However, I didn't think the 20% faster AppVeyor builld was worth dropping the Race Detector from the tests runs on Windows.

Reply:
> The Race Detector relies on CGO

Please add code comment for that?